app/editor/src/store/hooks/lookup/utils/filterEnabled.ts

```diff
@@ -0,0 +1,3 @@
+export const filterEnabled = (data: any[]) => {
```

Comment:
We should document our methods more often.  I'm guilty of the same in javascript presently.  Trying to get work done quickly.

Reply:
Will add some in now